src/core/collection/Collection.tsx

```diff
@@ -337,11 +350,13 @@ export function getLocationsCollectionCards({
   getRoutedInternalHref,
   getLocationCardProps,
   VenueCardContent,
+  CheckMarkIcon,
 }: {
   items: VenueType[];
   getRoutedInternalHref: Config['utils']['getRoutedInternalHref'];
   getLocationCardProps: Config['utils']['getLocationCardProps'];
   VenueCardContent: React.FC<Record<string, unknown>>;
+  CheckMarkIcon: React.FC<Record<string, unknown>>;
```

Comment:
I think that this CheckMarkIcon should be renamed to "HelsinkiCityOwnedIcon". If at some point we would get for example a Helsinki-Icon that we should use there, it would not be a checkmark anymore. Same comment here: https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/539

Reply:
renamed